controllers/controllers.js

```diff
@@ -52,15 +52,17 @@ exports.patchReview = (req, res, next) => {
 exports.getAllReviews = (req, res, next) => {
   const sort_by = req.query.sort_by;
   const order_by = req.query.order_by;
-  const category = req.query.category;
+  const category = [];
+
+  for (const [key, value] of Object.entries(req.query)) {
+    if (key !== "sort_by" && key !== "order_by") {
+      category.push(value);
+    }
+  }
```

Comment:
this logic should be handled in the model.  the job of the controller is simply to pass on all the arguments into the model function and then send the return value back to our user